frontend/src/metabase/models/components/ModelDetailPage/ModelDetailPage.styled.tsx

```diff
@@ -29,6 +29,10 @@ export const TabList = styled(BaseTabList)`
     display: flex;
   }
 
+  ${TabLink.Root}[aria-selected="false"] {
+    color: ${color("text-dark")};
+  }
```

Comment:
I don't see tab link being used in any other component except `<ModelDetailPage />` yet, what do you think about setting `TabLinkRoot` color to `text-dark` instead of declaring it here? I'm not even sure what `color: ${color("text-medium")};` set in `<TabLinkRoot />` was supposed to be for anyway. Like is there any text in `TabLink` that uses that color?

Reply:
That's right, I've used `text-medium` for `TabLink` by default, so it matches the styling of regular `Tab`
I think it's worth keeping them consistent, but I don't have a strong opinion on that

Reply:
This makes me think about @nemanjaglumac 's take on our app being paler lately. I'd think rather have the default color in the `TabLink` being `text-dark`. But looping @kdoh in for more opinions.

Also, if I'm not mistaken, from a quick search seems like `<Tab />` is used in only one place the `<BulkFilterModal />` so in total we use:
- `Tab` in `BulkFilterModal`
- `TabLink` in `ModelDetailPage`

I think it's worth trying to make these 2 components' colors more consistent.

#### <BulkFilterModal />
<img width="240" alt="image" src="https://user-images.githubusercontent.com/1937582/216651679-ebaca097-08da-49bf-8524-c31e23e28533.png">


#### <ModelDetailPage />
<img width="292" alt="image" src="https://user-images.githubusercontent.com/1937582/216651764-6e1fd5c0-a358-46d9-b948-a7dfdb805723.png">

Reply:
Yeah, perhaps `BulkFilterModal` tabs could also use a higher contrast with `text-dark`. Appreciate the investigation @WiNloSt 🙏

Reply:
Just to close the loop here, very interested in increasing the contrast of this component and upping our game there elsewhere too. 👓 👍